docs/metrics-events.md

```diff
@@ -264,6 +264,11 @@ to each of the table names mentioned above:
 
 ## Significant changes
 
+### Train 86
+
+* [The `sms.${templateName}.sent event
+  was fixed](https://github.com/mozilla/fxa-auth-server/pull/1843).
```

Comment:
This message is kinda vague, the PR title seems like it would be a perfect one liner.